MUXSDKKaltura/MUXSDKKaltura/MUXSDKPlayerBinding.swift

```diff
@@ -395,15 +390,17 @@ public class MUXSDKPlayerBinding: NSObject {
         self.videoData.hasUpdates = false
     }
     
-    private func handleRenditionChange(bitrate: Double) {
-        guard self.videoData.lastAdvertisedBitrate != 0 else {
-            // Starting Playback
+    private func handleRenditionChange(bitrate: Double, playbackStartDate: Date?) {
+        guard
+            self.videoData.lastAdvertisedBitrate != 0,
+            self.videoData.started
+        else {
             self.videoData.lastAdvertisedBitrate = bitrate
```

Comment:
curious why we don't need to call `self.videoData.hasUpdates = true` here before returning since we do update it

Reply:
Also how does this track the very first rendition? We should be sending to the server the indicated bitrate for the last event before playback starts since we are assuming this is the first rendition